<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Attendance System</string>
    </property>
   </widget>
   <!-- <widget class="QPushButton" name="newDateButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>New Date</string>
    </property>
   </widget> -->
   <widget class="QLineEdit" name="recordField">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type Student Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="insertField">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>New Student Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordTimeButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addStudentButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retrainButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrain Models</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
